src/fragments/lib/geo/native_common/getting_started/common.mdx

```diff
@@ -0,0 +1,36 @@
+<Callout>
+
+**Note:** Amplify Geo is in developer preview and is not intended to be used in production environments. Please reach out to us for any feedback and/or issues [here](https://github.com/aws-amplify/amplify-android/issues)
```

Comment:
Since this file is in native_common, I don't think we should have an Android-specific link in this file.

Reply:
good catch!